src/components/MerchantsPage/gam/GiftAMealPage.tsx

```diff
@@ -87,7 +87,10 @@ const GiftAMealPage = (props: Props) => {
         <CampaignListItem campaign={campaign} />
       ))}
 
-      <VideoComponent videoId="FYlUOhzYlRM"></VideoComponent>
+      <div className={styles.videoContainer}>
+        <VideoComponent videoId="FYlUOhzYlRM"></VideoComponent>
+        <h5 className={styles.videoDescription}>Watch how we bring impact to our communties through Gift-a-Meal.</h5>
```

Comment:
nit: should this be translatable too?

Reply:
Hmm yes thats on my TODO list for followups. Translating the GAM page will be a bit harder than other pages because some of the text (location, description, org names) come from the server, some other text is static.

Reply:
Makes sense, sounds good!